packages/openapi-to-graphql/test/example_api7.test.ts

```diff
@@ -170,13 +170,13 @@ test('Receive data from the subscription after creating a new instance', () => {
       })
 
     setTimeout(() => {
-      graphql(createdSchema, query2, null, null, {
+      graphql({ schema: createdSchema, source: query2, variableValues: {
         deviceInput: {
           name: `${deviceName}`,
           userName: `${userName}`,
           status: false
         }
-      })
+      } })
```

Comment:
This also looks a little bit strange to me as well. However, I'm just going to go ahead a approve this and move on.